packages/core-components/api-report-alpha.md

```diff
@@ -0,0 +1,73 @@
+    readonly 'sidebar.starredIntroText': 'Fun fact! As you explore all the awesome plugins in Backstage, you can actually pin them to this side nav.Keep an eye out for the little star icon (⭐) next to the plugin name and give it a click!';
+    readonly 'sidebar.recentlyViewedIntroText': 'And your recently viewed plugins will pop up here!';
+    readonly 'sidebar.dismiss': 'Dismiss';
+    readonly 'copyTextButton.tooltipText': 'Text copied to clipboard';
+    readonly 'simpleStepper.finish': 'Finish';
+    readonly 'simpleStepper.reset': 'Reset';
+    readonly 'simpleStepper.next': 'Next';
+    readonly 'simpleStepper.skip': 'Skip';
+    readonly 'simpleStepper.back': 'Back';
+    readonly 'errorPage.title': 'Looks like someone dropped the mic!';
+    readonly 'errorPage.subtitle': 'ERROR {{status}}: {{statusMessage}}';
+    readonly 'errorPage.goBack': 'Go back';
+    readonly 'errorPage.orPlease': '... or please ';
+    readonly 'errorPage.contactSupport': 'contact support';
+    readonly 'errorPage.isBug': 'if you think this is a bug.';
+    readonly 'emptyState.missingAnnotation.title': 'Missing Annotation';
+    readonly 'emptyState.missingAnnotation.actionTitle': 'Add the annotation to your component YAML as shown in the highlighted example below:';
+    readonly 'emptyState.missingAnnotation.readMore': 'Read more';
+    readonly 'emptyState.missingAnnotation.descriptionPrefix_one': 'The annotation ';
+    readonly 'emptyState.missingAnnotation.descriptionPrefix_other': 'The annotations ';
+    readonly 'emptyState.missingAnnotation.descriptionSuffix_one': ' is missing. You need to add the annotation to your component if you want to enable this tool.';
+    readonly 'emptyState.missingAnnotation.descriptionSuffix_other': ' are missing. You need to add the annotations to your component if you want to enable this tool.';
+    readonly 'supportConfig.title': 'Support Not Configured';
+    readonly 'supportConfig.links.title': 'Add `app.support` config key';
+    readonly 'errorBoundary.title': 'Please contact {{slackChannel}} for help.';
+    readonly 'oauthRequestDialog.title': 'Login Required';
+    readonly 'oauthRequestDialog.authRedirectTitle': 'This will trigger a http redirect to OAuth Login.';
+    readonly 'oauthRequestDialog.login': 'Log in';
+    readonly 'oauthRequestDialog.rejectAll': 'Reject All';
+    readonly 'supportButton.title': 'Support';
+    readonly 'supportButton.close': 'Close';
+    readonly 'alertDisplay.message_one': '({{ num }} older message)';
+    readonly 'alertDisplay.message_other': '({{ num }} older messages)';
+    readonly 'autoLogout.stillTherePrompt.title': 'Logging out due to inactivity';
+    readonly 'autoLogout.stillTherePrompt.description': 'You are about to be disconnected in';
+    readonly 'autoLogout.stillTherePrompt.second_one': 'second';
+    readonly 'autoLogout.stillTherePrompt.second_other': 'seconds';
+    readonly 'autoLogout.stillTherePrompt.descriptionSuffix': 'Are you still there?';
```

Comment:
Same as above again, this would be better solved with interpolation rather than seperate messages.

Reply:
You can remove these translations, and revert the components until we fix the interpolation feature. Same for `emptyState.missingAnnotation.descriptionPrefix*` ones